commerce-api/src/main/kotlin/com/hanghae/commerce/ErrorController.kt

```diff
@@ -10,16 +10,23 @@ import org.springframework.web.bind.annotation.RestController
 @RequestMapping("/api/raise")
 class ErrorController {
 
+
+
     @GetMapping("/exception")
     fun exception(): ResponseEntity<String> {
+
         throw IllegalArgumentException("api called - exception")
+
     }
 
     @GetMapping("/timeout/{seconds}")
     fun timeout(
         @PathVariable seconds: Long,
     ): ResponseEntity<String> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty